sources/osg/WebGLCaps.js

```diff
@@ -31,12 +73,16 @@ define( [
             return this._webGLParameters.MAX_SHADER_PRECISION_INT;
         },
         checkRTTSupport: function ( typeFloat, typeTexture ) {
+
             var gl = this._gl;
-            if ( gl === undefined )
+            if ( !gl )
                 return false;
             var key = typeFloat + ',' + typeTexture;
-            if ( this._checkRTT[ key ] !== undefined )
+
+            // check once only
+            if ( this._checkRTT[ key ] )
```

Comment:
k, thx, btw about using the  Boolean() instead ? advantage is that eslint does check for it, and it's primed for visibiliy 

 !!foo => Boolean(foo); http://eslint.org/docs/rules/no-implicit-coercion.html